### PR TITLE
🧑‍💻 552 - Add keycloak, mailhog to docker-compose

### PR DESCRIPTION
**Pull request by anncatton**
## Description of Changes

Adds Keycloak and Mailhog to the docker-compose. Uses Keycloak v22 bitnami docker image.

### PI-DAS, Consent DAS
- updates the `keycloakId`s of the PI Participant seed data to match the example users in the Keycloak container
- corrects some data in both DASes that was out of date with the current participant/guardian models

### Documentation
- adds a section for Keycloak and Mailhog in the docker-compose setup
- updates `.env.schema` files for `consent-ui` and `consent-api`

### Special Instructions

Add the required env vars from the `.env.schema` files for consent-ui and consent-api to connect to the local Keycloak instance. The applicable realm is called `local-platform`.
To test with the example users in the prepopulated in the `local-platform` realm, you will need to re-seed the `pi-das` and `consent-das` databases, as some of this data has been updated.

## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [ ] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [x] New environment variables added to `.env.schema` files, `README.md`
- [ ] Added copyrights to new files

### CODE REVIEW

**Review by mistryrn — COMMENTED** [review]
Looks good! Thanks for updating the seed data as well 🙇 

I'm gonna push a couple changes based on the directory renaming/restructuring I tested that we talked about on Slack, but in the meantime left a couple questions about some of the .env.schema values

File: apps/consent-ui/.env.schema [hunk 1]
```diff
@@ -14,16 +14,16 @@ AUTH_URL=http://localhost:3000/api/auth
 TOKEN_ENCRYPTION_KEY=encryptionsecretz
 TOKEN_MAX_AGE=3600
 # the server will start without these values, but Authentication flow will not work
-AUTH_KEYCLOAK_ISSUER=
```
[the remark this answers is not in the document]
Comment (anncatton) [reply apps/consent-ui/.env.schema:17]: i think it can be written with or without quotes

[the remark this answers is not in the document]
Comment (mistryrn) [reply apps/consent-ui/.env.schema:17]: Ah, cool. For consistency with our other URLs I'm gonna drop the quotes then 🙂

**Review by mistryrn — APPROVED** [review]
🚀

**Pull request merged**